frontend/src/Editor/Editor.jsx

```diff
@@ -105,6 +105,10 @@ class Editor extends React.Component {
     };
   }
 
+  setWindowTitle(name) {
+    document.title = name ? `Tooljet - ${name}` : `Tooljet - Untitled App`;
```

Comment:
Can we reverse the order? `<app name> - ToolJet`?

Reply:
Sure